code/modules/modular_computers/file_system/programs/ntdownloader.dm

```diff
@@ -115,6 +123,7 @@
 
 	var/list/data = get_header_data()
 
+	data["theme"] = theme
```

Comment:
Can you make it somewhat consistent, because you have an extra `theme` in addition to `PC_device_theme`? It's duplicate functionality, make it only `PC_device_theme`.

Reply:
The reason I have these split is that I'm using `theme` when it's being set due to the program, and `PC_device_theme` when it's being set due to the type of computer. The downloader, file manager, and hardware config apps are all "built-in" apps (in that they cannot be deleted, transferred, or downloaded), and so ought to be following the theme set by the device. Contrast to individual apps like Fission360, which can be used on an NT device if it's been emagged.

The long and the short of it is that by using Fission360 on an NT tablet, you'd have `device_theme = "ntos"` but also `theme = "syndicate"`. The former would change the system tray text to say NtOS, while the latter tells the app how to look.

Reply:
The big block of text here I meant to add to the first change request and got mixed up.

ntdownloader now uses `PC_device_theme`, as do the hardware config and file manager.